src/instantsearch/instantsearch.ts

```diff
@@ -122,6 +122,13 @@ export interface SearchForFacetValuesRequestParameters
 export type GeoRectangle = [number, number, number, number];
 export type GeoPolygon = [number, number, number, number, number, number];
 
+export type FacetSortByStringOptions =
+  | 'count:asc'
```

Comment:
We can also provide `name | count`.

Reply:
I wasn't aware! I see this now in the helper.
Do you think it makes sense to add them although they are not documented in [InstantSearch docs ? ](https://www.algolia.com/doc/api-reference/widgets/menu-select/js/#widget-param-sortby) Shall we add them there too ?

Reply:
I wasn't sure about the doc, it's still good to add them since it's available.

Reply:
ok

Reply:
done here: c990999